app.rb

```diff
@@ -0,0 +1,129 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # Create a book
+  def create_book
+    print 'Title:'
+    title = gets.chomp
+    print 'Author:'
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books.push(book)
+    puts 'Book created successfully'
+  end
+
+  # List all books
+  def list_books
+    if @books.empty?
+      puts 'No books in the library'
+      return
+    end
+    @books.each do |singlebook|
+      puts "Title: #{singlebook.title}, Author: #{singlebook.author}"
+    end
+  end
+
+  # List people
+  def list_people
+    if @people.empty?
+      puts 'No person in library'
+      return
+    end
+    @people.each do |person|
+      puts "[#{person.class}], Name:#{person.name}, ID:#{person.id}, Age:#{person.age}"
+    end
+  end
+
+  # Create student
+  def create_student
```

Comment:
- [ ] Hi @Graycemuthui, fantastic job so far on implementing this method. Let's also make sure that permission should be granted to a student with an age less than 18 before being created. Once more, let's kindly watch this [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).